pool/web/handler/admin_experiments_waiting_list.ml

```diff
@@ -172,14 +175,17 @@ let assign_contact req =
       let session_text = Session.(to_email_text language session) in
       Lwt_result.return Email.{ subject; text; language; session_text }
     in
+    let tags = Logger.req req in
     let events =
       Cqrs_command.Assignment_command.CreateFromWaitingList.(
-        handle { session; waiting_list; already_enrolled })
+        handle ~tags { session; waiting_list; already_enrolled })
```

Comment:
Did you check that `tags` are passed in for every single command handler?

Reply:
root_command.ml needs to be done